dcrlibwallet.go

```diff
@@ -2,70 +2,58 @@ package dcrlibwallet
 
 import (
 	"context"
-	"fmt"
-	"os"
 	"path/filepath"
 
-	"github.com/asdine/storm"
-	"github.com/decred/dcrwallet/errors"
-	"github.com/decred/dcrwallet/netparams"
-	"github.com/decred/dcrwallet/wallet"
-	"github.com/decred/dcrwallet/wallet/txrules"
+	"github.com/decred/dcrwallet/errors/v2"
+	"github.com/decred/dcrwallet/wallet/v3"
+	"github.com/decred/dcrwallet/wallet/v3/txrules"
+	"github.com/raedahgroup/dcrlibwallet/internal/loader"
+	"github.com/raedahgroup/dcrlibwallet/internal/netparams"
 	"github.com/raedahgroup/dcrlibwallet/txindex"
 	"github.com/raedahgroup/dcrlibwallet/utils"
-	"go.etcd.io/bbolt"
 )
 
-const logFileName = "dcrlibwallet.log"
+type Properties struct {
+	ID                     int    `storm:"id,increment"`
+	Name                   string `storm:"unique"`
+	DataDir                string
+	Seed                   string
+	SpendingPassphraseType int32
+	DiscoveredAccounts     bool
+}
 
 type LibWallet struct {
-	walletDataDir string
-	activeNet     *netparams.Params
-	walletLoader  *WalletLoader
-	wallet        *wallet.Wallet
-	txDB          *txindex.DB
-	configDB      *storm.DB
-	*syncData
+	*Properties `storm:"inline"`
```

Comment:
The `storm:"inline"` meta info should no longer be necessary as this `LibWallet` struct isn't being saved to the db.